lib/spack/docs/getting_started.rst

```diff
@@ -1580,6 +1590,13 @@ as the project providing Git support on Windows. This is additionally the recomm
 for installing Git on Windows, a link to which can be found above. Spack requires the
 utilities vendored by this project.
 
+"""
+7z
+"""
+
+A tool for extracting ``.xz`` files is required for extracting source tarballs. 7z
+can be downloaded from the Windows Store.
```

Comment:
We should point users to either Spack, or the official sourceforge distribution of 7zip, not the Windows Store. This one isn't as problematic as Python from the store, but it still an added point of complexity I'd rather not actively advocate for.